PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -274,7 +282,7 @@ struct HfTaskFlowCharmHadrons {
   /// Compute the scalar product
   /// \param collision is the collision with the Q vector information and event plane
   /// \param candidates are the selected candidates
-  template <int DecayChannel, typename T1>
+  template <DecayChannel Channel, typename T1>
```

Comment:
```suggestion
  template <DecayChannel channel, typename T1>
```
As I explained before, `DecayChannel` is a type (therefore starts with a capital letter), `channel` is a variable (therefore starts with a lower-case letter).